src/redux/books/books.js

```diff
@@ -1,54 +1,67 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] [OPTIONAL] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃